login/index.js

```diff
@@ -0,0 +1,72 @@
+const inputEmail = document.querySelector('#login-email');
+const inputPassword = document.querySelector('#login-password');
+const loginButton = document.querySelector('#login-button');
+
+inputEmail.addEventListener('focusout', noInput);
+inputEmail.addEventListener('focusout', emailCheck);
+inputEmail.addEventListener('keyup', enterKey);
+
+inputPassword.addEventListener('focusout', noInput);
+inputPassword.addEventListener('keyup', enterKey);
+
+loginButton.addEventListener('click', correctInput);
+
+
+// input 없을 때 에러 메세지
+function noInput(e) {
+  if (e.target.value === '') {
+
+    if (e.target.classList.contains('login-email')) {
+      e.target.nextElementSibling.textContent = '이메일을 입력해 주세요.';
+      e.target.style.border = '1px solid #FF5B56';
+    }
+    else if (e.target.classList.contains('login-password')) {
+      e.target.nextElementSibling.textContent = '비밀번호를 입력해 주세요.';
+      e.target.style.border = '1px solid #FF5B56';
+    }
+
+  }
+}
+
+
+// 이메일 유효성 확인
+function emailValidation(input) {
+  let reg_email = /^([0-9a-zA-Z_\.-]+)@([0-9a-zA-Z_-]+)(\.[0-9a-zA-Z_-]+){1,2}$/;
```

Comment:
- 사용하는 변수명은 camelCase로 변경하는게 좋을 것 같아요.
- 변경되지 않는 값의 경우 `let`보다는 `const`를 이용해 선언하는게 좋을 것 같아요.
```suggestion
  const regEmail = /^([0-9a-zA-Z_\.-]+)@([0-9a-zA-Z_-]+)(\.[0-9a-zA-Z_-]+){1,2}$/;
```